### DISCUSSION TITLE
star rating tooltips from 7*-8* is difficult to read

### DISCUSSION THREAD

**Discussion by iSimpFor** [ppy/osu#15508] [Ideas] [answered]
![osu_2021-11-07_01-50-47](https://user-images.githubusercontent.com/44514667/140635515-05bc3db8-33b1-4d49-b570-9da1514e4141.png)
shouldn't they also be gold colored?
version: 2021.1105.0

**Comment by Joehuu** [comment, marked as answer]
The panel design is not updated yet to match web, see https://github.com/ppy/osu/issues/14216. The new design doesn't use tooltips, hovering expands the panel instead.

**Reply by frenzibyte** [reply]
Given that the implementation for the new design is around the corner, this can remain as-is for the time being.
